src/vault/env.clj

```diff
@@ -90,3 +120,13 @@
        (resolve-secrets client env secrets))
      ; No secrets, return env directly.
      env)))
+
+
+(defn write!
```

Comment:
I would not add these to the `vault.env` namespace - caching should be a feature in the client itself, which would keep us from needing to put the `write!` function here.